lib/averages.js

```diff
@@ -7,4 +7,13 @@ function total(values) {
   return total;
 };
 
-module.exports = total;
+function median(values) {
```

Comment:
you're using some calculations below more than once. it'd be cleaner/easier to read if you set those as variables so you don't have to evaluate them more than once 👍